docs/DeveloperGuide.md

```diff
@@ -4,14 +4,15 @@ title: Developer Guide
 ---
 
 * Table of Contents
-  {:toc}
+{:toc}
 
 --------------------------------------------------------------------------------------------------------------------
 
 ## **Acknowledgements**
 
-* {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the
-  original source as well}
+| Module                                  | Description                               |
+|-----------------------------------------|-------------------------------------------|
+| [ZXing](https://github.com/zxing/zxing) | Barcode image processing library for Java |
```

Comment:
Should we include JavaFX, Jackson & JUnit5 here too? Similar to the one in the index file.

```suggestion
| [JavaFX](https://openjfx.io/)                   | UI generation for Java |
| [Jackson](https://github.com/FasterXML/jackson) | Json processing library for Java |
| [JUnit5](https://github.com/junit-team/junit5)  | Automated testing library for Java |
| [ZXing](https://github.com/zxing/zxing)         | Barcode image processing library for Java |
```

Reply:
Good idea